docs/INSTALL.rst

```diff
@@ -6,43 +6,54 @@ Installation instructions
   issues for a variety of platforms. See README for the general instruction
   manual.
 
-1) Linux on x86
----------------
+Platforms
+=========
+
+Linux on x86
+------------
 
 This platform is expected to work well. Compile the program with:
 
-$ make
+.. code-block:: console
+
+  $ make
 
 You can start using the fuzzer without installation, but it is also possible to
 install it with:
 
-# make install
+.. code-block:: console
+
+  # make install
 
 There are no special dependencies to speak of; you will need GNU make and a
 working compiler (gcc or clang). Some of the optional scripts bundled with the
 program may depend on bash, gdb, and similar basic tools.
 
-If you are using clang, please review llvm_mode/README.llvm; the LLVM
+If you are using clang, please review `llvm_mode/README.llvm`; the LLVM
```

Comment:
Make llvm_mode/README.llvm a link?

Reply:
Gonna leave this as it is for now, gotta think about what to do here since these individual READMEs like `llvm_mode/README.llvm` and `qemu/README.qemu` live under their own directories and not the docs folder.